index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
-says "Send a termination request for the presentation to its receiving user
-agent using an implementation specific mechanism", the [=controlling user
+When
+[[PRESENTATION-API#terminating-a-presentation-in-a-controlling-browsing-context|section
+6.5.6]] says "Send a termination request for the presentation to its receiving
+user agent using an implementation specific mechanism", the [=controlling user
 agent=] may send a presentation-termination-request message.
 
-When [[PRESENTATION-API|Presentation API]] [section
-6.7.1](https://www.w3.org/TR/presentation-api/#monitoring-incoming-presentation-connections)
+When [[PRESENTATION-API#monitoring-incoming-presentation-connections|section
+6.7.1]]
 says "it MUST listen to and accept incoming connection requests from a
 controlling browsing context using an implementation specific
 mechanism", the [=receiving user agent=] must receive and process the
 presentation-connection-open-request.
 
-When [[PRESENTATION-API|Presentation API]] [section
-6.7.1](https://www.w3.org/TR/presentation-api/#monitoring-incoming-presentation-connections)
-says "Establish the connection between the controlling and receiving browsing
-contexts using an implementation specific mechanism.", the [=receiving user
-agent=], must send a presentation-connection-open-response message.
+When [[PRESENTATION-API#monitoring-incoming-presentation-connections|section
+6.7.1]] says "Establish the connection between the controlling and receiving
+browsing contexts using an implementation specific mechanism.", the [=receiving
+user agent=], must send a presentation-connection-open-response message.
 
 
-Remote Playback API Protocol {#remote-playback}
------------------------------------------------
+Remote Playback Protocol {#remote-playback-protocol}
+----------------------------------------------------
 
-Issue(12): Propose control protocol for Remote
-Playback API.
+This section defines the use of the Open Screen Protocol for starting, terminating,
+and controlling remote playback of media as defined by the
+[[REMOTE-PLAYBACK|Remote Playback API]].  [[#remote-playback-api]] defines how
+APIs in [[REMOTE-PLAYBACK|Remote Playback API]] map to the protocol messages
+defined in this section.  
+ 
+For all messages defined in this section, see Appendix A for the full
+CDDL definitions.
+
+<!-- TODO: Add a capability that indicates support for the
+remote playback protocol.
+See https://github.com/webscreens/openscreenprotocol/issues/123 -->
+<!-- TODO: Frequency to update current-time (HtmlMediaElement does every 250ms?) -->
+<!-- TODO: extended mime types in source tag? -->
+<!-- TODO: media queries replacing URLs? -->
+
+To learn which receivers are [=compatible remote playback device=]s (also called
+available [=remote playback devices=]) for a particular URL or set of URLs, the
+controller may send a remote-playback-availability-request message with the
+following values:
+
+: urls
```

Comment:
Please correct me if I'm wrong, but wouldn't we always be querying with a single URL (URL of the media element)?
https://w3c.github.io/remote-playback/#dom-remoteplayback-watchavailability